File: test_cases/python/claude/11.py
def run_tests(solution_module):
    f = solution_module.minWindow
    # t is single char repeated — needs both
    assert f('aab', 'ab') == 'ab'
    # no valid window
    assert f('abc', 'd') == ''
    # t longer than s
    assert f('a', 'aa') == ''
    # entire string is minimum window
    assert f('bca', 'abc') == 'bca'
    # window shrinks from wider match
    assert f('acbbaca', 'aba') == 'baca'

File: test_cases/c/claude/11.c
#include <stdio.h>
#include <string.h>
#include <assert.h>

char* minWindow(char* s, char* t);

int main(void) {
    assert(strcmp(minWindow("ADOBECODEBANC", "ABC"), "BANC") == 0);
    assert(strcmp(minWindow("a", "a"), "a") == 0);
    assert(strcmp(minWindow("a", "aa"), "") == 0);
    assert(strcmp(minWindow("aa", "aa"), "aa") == 0);
    assert(strcmp(minWindow("aAbBcC", "ABC"), "ABcC") == 0);
    return 0;
}